StoreManager UI

NabunyaLilian opened the pull request:
I have completed the UI pages and this is my first pull request and any feedback is greatly appreciated.
Thanks
@amcodex, @armstrongsouljah, @RLubwama, @masete, @nuwagalb

NabunyaLilian committed:
Add  login feature [delivers #161112568]

NabunyaLilian committed:
[feature #161112568] Add  login feature

NabunyaLilian committed:
[feature #161112568] Add  login feature

NabunyaLilian committed:
[#161113316,#161113486,#161112959,#161178551]add manage products feature

NabunyaLilian committed:
[#161113128]add creating new sale attendant user account feature

NabunyaLilian committed:
[#161112735,#161112853]add manage sale record feature

NabunyaLilian committed:
[#161113622]view sales by attendants feature

NabunyaLilian committed:
[#161112959]view products feature

NabunyaLilian committed:
[#161112687]add to cart feature

NabunyaLilian committed:
[#161112959]updated view products feature

NabunyaLilian committed:
Add readme and changes

NabunyaLilian committed:
delete files outside ui

NabunyaLilian committed:
add readme

NabunyaLilian committed:
New changes added

NabunyaLilian committed:
validation added

NabunyaLilian committed:
Merge pull request #1 from NabunyaLilian/gh-pages

Created UI of StoreManager

armstrongsouljah commented:
Hey @NabunyaLilian could paste a link to your `gh-pages`

ghost commented:
Li, add link to gh-pages when pull requesting again

NabunyaLilian commented:
https://nabunyalilian.github.io/StoreManager/UI/login.html that the link to github pages admin credentials ;
admin and secret then storeattendant Lia and password

nuwagalb commented on admin index.html:
```diff
@@ -0,0 +1,119 @@
+  <div>
+  
+  <ul>
+  <img  src = "img/logo.png" style="float: left;position: relative;height:10px ;height :45px">
```
Avoid using inline css. Add this code to an external css file

nuwagalb commented on admin index.html:
```diff
@@ -0,0 +1,119 @@
+  <h1 id = 'heading'>Products</h1> 
+<section>
+ <table>
+  <button id="modalBtn" class="button" style = 'float:right; margin-right: 138px ;margin-top: -70px'> + Add product</button>
```
Avoid using inline css. Add this code to an external css file

masete commented:
Nice work lillian, i like your page redirection of pages using javascript, but work on your responsiveness to the size of a phone and a tablet.

NabunyaLilian commented:
@masete thanks

nuwagalb commented on accounts.html:
```diff
@@ -0,0 +1,34 @@
+  <div>
+  
+  <ul>
+  <img  src = "img/logo.png" style="float: left;position: relative;height:10px ;height :45px;">
```
Avoid using inline css. Find a way of adding it to your external css file

nuwagalb commented on accounts.html:
```diff
@@ -0,0 +1,34 @@
+  <li><a href="#">Products</a></li>
+  <li><a class="active">Accounts</a></li>
+  <li><a href="#">Sales</a></li>
+  <li style="float:right"><a href="#">Log out</a></li>
```
remove this inline css and add it to your external css file

nuwagalb commented on UI/accounts.html:
```diff
@@ -0,0 +1,69 @@
+  <div>
+  
+  <ul>
+  <a href="admin_index.html"><img  src = "img/logo.png" style="float: left;position: relative;height:10px ;height :45px "></a>
```
avoid inline css. add it to your external css file

nuwagalb commented on UI/accounts.html:
```diff
@@ -0,0 +1,69 @@
+  <li><a href="admin_index.html">Products</a></li>
+  <li><a class="active" href="#">Accounts</a></li>
+  <li><a href="sales.html">Sales</a></li>
+  <li style="float:right"><a href="login.html">Log out</a></li>
```
avoid inline css. add it to your external css file

NabunyaLilian replied:
@nuwagalb thanx i will correct it

NabunyaLilian committed:
validation added

NabunyaLilian committed:
validation added

NabunyaLilian committed:
new modifications

NabunyaLilian committed:
Merge pull request #8 from NabunyaLilian/feature

Merge pull request #1 from NabunyaLilian/gh-pages

NabunyaLilian committed:
new changes

NabunyaLilian committed:
new changes in sales

NabunyaLilian committed:
new changes in sales

NabunyaLilian committed:
feedback modifications

NabunyaLilian committed:
Merge branch 'gh-pages' of https://github.com/NabunyaLilian/StoreManager into gh-pages

NabunyaLilian committed:
feedback modifications

NabunyaLilian committed:
git merge conflict

NabunyaLilian committed:
update UI

NabunyaLilian committed:
[#162158216] add login frontend

NabunyaLilian committed:
Update Challenge 4
- create js files for frontend
- set product.js to handle a specific product
- set products.js to handle products interface
- set sales.js to handle sales interface
- set sale.js to handle a specific sales
- set view_product.js to handle viewing of products

NabunyaLilian committed:
[#162158531,#162158422,#162158381,#162158345,#162158301,#162158267,#162158241]update files

NabunyaLilian committed:
update readme

NabunyaLilian committed:
update js files to work with heroku

NabunyaLilian committed:
update js files to work with heroku

NabunyaLilian committed:
update gitignore

NabunyaLilian committed:
Merge pull request #14 from NabunyaLilian/frontend

Frontend
